### ISSUE TITLE
Problems creating symlinks with Windows 7 host and Centos 6.5 guest

### ISSUE THREAD

**Issue by pablogosse** [mitchellh/vagrant#5840]
Hi all. I'm encountering a problem with Vagrant when trying to create symlinks on the guest in the folder that is shared with the host (/vagrant) and I'm hoping someone can help. Here's my environment info:

**Host OS:** Windows 7 Enterprise SP 1
**Guest OS:** CentOS release 6.5 (Final) (using the chef/centos-6.5 box)
**VMWare Workstation:** 11.1.0
**Vagrant:** 1.7.2
**Vagrant Plugins:** vagrant-share (1.1.3, system); vagrant-vmware-workstation (3.2.8)

I'm trying to create a symlink inside the /vagrant folder such that files can be added to it from the Windows host.

In my Vagrantfile I have the following:

    config.vm.provider "vmware_workstation" do |v|
        v.vmx["sharedFolder0.followSymlinks"] = "TRUE"
    end

When I try to create a symlink inside the /vagrant shared folder, this is the result:

    [root@drupal-dev-gossep-local ~]# mkdir /var/derp
    [root@drupal-dev-gossep-local ~]# ln -s /var/derp /vagrant/derplink
    ln: creating symbolic link `/vagrant/derplink': Operation not supported

Debug output from both vagrant up and vagrant reload show the following:

   INFO vmx_modify:   - Set: sharedFolder0.followSymlinks = 'TRUE'

However when I manually inspect the vmx file after the machine is up, I see:

    sharedfolder0.followsymlinks = "FALSE"

Even if I change the vmx file to TRUE, when I reboot the machine it is reset to FALSE.

I saw this post (https://github.com/mitchellh/vagrant/issues/4809) in which Mitchell indicates that this should be fixed now, but I am still experiencing the behavior described in the final comment - https://github.com/mitchellh/vagrant/issues/4809#issuecomment-102632207

Can anyone provide any advice?

Thanks,

Pablo

**Comment by nfloersch** [comment]
Check this out... I think it applies to your setup.

http://blog.prolificinteractive.com/2015/01/21/getting-vagrant-nodejs-windows-play-well-together/

**Comment by nfloersch** [comment]
I agree that this bug is still happening with VMWare Workstation v11 on the host. My host is a Windows 8.1 machine. The Vagrant VM is a Windows Server 2012 R2 machine.

When I put the following in my Vagrantfile...

> config.vm.provider "vmware_workstation" do |v|
>    #v.gui = true
>    **v.vmx["sharedFolder0.followSymlinks"] = "TRUE"**
> end

I get a VMX that has ...

> [...]
> scsi0:0.filename = "disk.vmdk"
> scsi0:0.present = "TRUE"
> scsi0:0.redo = ""
> sharedfolder.maxnum = "1"
> **sharedfolder0.followsymlinks = "FALSE"**
> softpoweroff = "FALSE"
> sound.startconnected = "FALSE"
> tools.synctime = "TRUE"
> tools.upgrade.policy = "upgradeAtPowerCycle"
> [...]

**Comment by nfloersch** [comment]
Interesting ... I just noted that my VMX file was for VMWare Workstation v9.0 virtual hardware. 
So I upgraded it to version 11.

Now if I manually edit the VMX file and add the line:

> sharedfolder0.followsymlinks = "TRUE"

VMWare Workstation refuses to load or open the file, stating that it is corrupt. 
The same happens if I set the value to FALSE, also.

So while it appears that the Vagrant code is doing the exact opposite of what we want in putting a value of "FALSE" in where we told it to put a "TRUE" ... or perhaps just leaving the line out of the VMX altogether ... ultimately it appears that putting anything in the VMX for a newer virtual hardware machine may just break it? So Vagrant is being smart..?
